clients/python-wrapper/lakefs/client.py

```diff
@@ -5,22 +5,29 @@
 """
 
 from __future__ import annotations
-
+import os
 import base64
 import json
 from threading import Lock
 from typing import Optional
 from typing import TYPE_CHECKING
 from urllib.parse import urlparse, parse_qs
+import boto3.session
 
 import lakefs_sdk
 from lakefs_sdk import ExternalLoginInformation
 from lakefs_sdk.client import LakeFSClient
 
 from lakefs.config import ClientConfig
-from lakefs.exceptions import NotAuthorizedException, ServerException, api_exception_handler
+from lakefs.exceptions import NotAuthorizedException, ServerException, NoAuthenticationFound, api_exception_handler
 from lakefs.models import ServerStorageConfiguration
 
+from .config import (
```

Comment:
Please use the same coding conventions we have now:
```suggestion
from lakefs.config import (
```